mainnet.env

```diff
@@ -0,0 +1,113 @@
+###############################################################################
+#                                ↓ REQUIRED ↓                                 #
+###############################################################################
+
+# Network to run the node on ("mainnet","alfajores" or "baklava")
+NETWORK_NAME=mainnet
+
+# Type of node to run ("full" or "archive"), note that "archive" is 10x bigger
+NODE_TYPE=full
+
+###############################################################################
+#                            ↓ REQUIRED (BEDROCK) ↓                           #
+###############################################################################
+
+# L1 node that the op-node (Bedrock) will get chain data from
+OP_NODE__RPC_ENDPOINT=https://ethereum-rpc.publicnode.com
```

Comment:
Should we add a comment here to mention that people should change this

Reply:
Do they need to change that? I thought this is fine.

Reply:
I guess we'd recommend to use a RPC provider they trust + have an account without rate-limiting or throttling 🤔

Reply:
Or their own node

Reply:
Ok, I was thinking that wold be self explanatory for node operators, but I can add it.

Reply:
Please add it, then it's written down ;)